### PR TITLE
Changes based on ID review

### PR DESCRIPTION
**Pull request by dmorgant**
**ID feedback**

- [x] Generally we hyphenate compound adjectives, such as "fault-tolerant" (as in "fault-tolerant microservices"), unless we decided that we don't want to hyphenate that across all of OL. :)

**What you'll learn**
- [x] In the second paragraph, I'd simplify "to enable you to use" to just "to use" 
- [x] Clarify causation in the sentence after that:
"You'll then add the mpFaultTolerance-1.0 feature to the server.xml file to use the MicroProfile Fault Tolerance feature. Next, you'll include the @CircuitBreaker annotation, which causes your microservice to fail immediately and prevents repeated calls that are likely to fail."
--> We reviewed this comment with the team and felt no change was needed.
- [x] I'd also simplify/omit passive voice in the last sentence:
"Finally, you will add a fallback service to invoke an alternate method for a failing service."

**Configuring the failure threshold** suggested changes:
- [x] "When the circuit is open, calls to the failing service fail immediately."
- [x] "To determine the failure threshold, the policy monitors a rolling window, a specified number of the most recent consecutive requests made to the service."
- [x] "The minimum number of requests that the program must make before the parameter can determine whether the circuit trips. This parameter also specifies the size of the rolling window."
--> 'program' does not work here and the 'parameter' doesn't determine...it is **used** in determining.  We changed this to 'The minimum number of requests that must be made to the service before determining whether the circuit trips.  This parameter also specifies the size of the rolling window.'
- [x] "To determine the minimum number of failure requests necessary within a rolling window to trip the circuit, multiply the requestVolumeThreshold parameter by the failureRatio parameter. The total requests must be at least the size of the rolling window before the circuit breaker can trigger."

After following the green box instructions:
- [x] "The request is routed to the Check Balance microservice**,** but the microservice is down." <--Note the comma before "but." :)
- [x] "The circuit breaker policy opens the circuit after one failure, which comes from multiplying the requestVolumeThreshold parameter (2) by the failureRatio parameter (0.5). However, the circuit remains closed because the number of requests is fewer than the size of the rolling window (2)." <--"Fewer" is generally used with countable things, and "less" is used for things that can't be counted.

- [x] In the text below the "Reset" button, could we remove the parenthesis?
 "Connection attempts from most recent to oldest. The brackets represent the rolling window:"

**Delay definition**
- [ ] "The delay in milliseconds after the circuit opens until checking the availability of the main service. The default is 5000 ms."
What checks the availability of the main service? My guess is the circuit...but if not, I think we should specify what performs the check, such as, "...after the circuit opens until the <thing> checks the availability..."
--> We decided not to change anything here.

**successThreshold**
- [x] Our grammar checker really likes relative pronouns. I think this helps with translation (in case we decide to translate guides later):
"The number of consecutive successful invocations of the service **that are** required before the circuit is closed. The default is 1 request."

### CODE REVIEW

File: json-guides/circuit-breaker.json [hunk 1]
```diff
@@ -303,10 +303,10 @@
...
-              "<li><b>requestVolumeThreshold</b>: The minimum number of requests that must be made before determining if the circuit should trip. This also specifies the size of the rolling window. The default is <code>20</code> requests.",
+              "<li><b>requestVolumeThreshold</b>: The minimum number of requests that must be made to the service before determining whether the circuit trips. This parameter also specifies the size of the rolling window. The default is <code>20</code> requests.",
               "<li><b>failureRatio</b>: The minimum failure ratio in the rolling window to trigger the circuit breaker. The default ratio is <code>0.5</code>.",
-              "<li><b>delay</b>: The delay in milliseconds to transition to a half-open state after the circuit opens. The default is <code>5000</code> ms.",
-              "<li><b>successThreshold</b>: The number of consecutive successful invocations of the service that is required before the circuit closes. The default is <code>1</code> request.",
+              "<li><b>delay</b>: The delay in milliseconds after the circuit opens to transition to a half-open state. The default is <code>5000</code> ms.",
```
Comment (kevinwlau) [inline json-guides/circuit-breaker.json]: The `delay` definition should also be changed in the step we introduce it.

File: json-guides/circuit-breaker.json [hunk 2]
```diff
@@ -303,10 +303,10 @@
               "Now that you learned about circuit breakers and fallbacks, you can explore the parameters in the <code>@CircuitBreaker</code> annotation and see the circuit in action.",
               "<br>You learned about the following parameters:",
               "<ul>",
-              "<li><b>requestVolumeThreshold</b>: The minimum number of requests that must be made before determining if the circuit should trip. This also specifies the size of the rolling window. The default is <code>20</code> requests.",
+              "<li><b>requestVolumeThreshold</b>: The minimum number of requests that must be made to the service before determining whether the circuit trips. This parameter also specifies the size of the rolling window. The default is <code>20</code> requests.",
```
Comment (kevinwlau) [inline json-guides/circuit-breaker.json]: `This parameter` is used here, while `The parameter` is used in the step we first introduce requestVolumeThreshold.

File: json-guides/circuit-breaker.json (hunk 1 above)
Comment (dmorgant) [reply json-guides/circuit-breaker.json]: Changed

File: json-guides/circuit-breaker.json (hunk 2 above)
Comment (dmorgant) [reply json-guides/circuit-breaker.json]: Changed.

**Comment by sishida** [comment]
Thanks so much for adding these changes. I found a couple more small things if it's still possible to add them in!

Configuring the failure threshold
I ran this through my grammar checker, and it'd prefer to make some changes to it:
"The minimum number of requests that must be made to the service before determining whether the circuit trips."
If we could please change the above text to this text below, that'd be great!
"The minimum number of requests that must be made to the service before it determines whether the circuit trips."

Also, I still don't see the comma here:
"The request is routed to the Check Balance microservice**,** but the microservice is down." <--Note the comma before "but."

**Pull request merged**